### PR TITLE
Clarify the expand_message implementation for ristretto255 and decaf448.

### PR DESCRIPTION
**Pull request by chris-wood**
This clarifies the change in #180.

cc @armfazh, @claucece

### CODE REVIEW

**Comment by claucece** [comment]
LGTM.

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -920,9 +920,10 @@ and ristretto255. See {{cryptanalysis}} for related discussion.
 
 - Group: ristretto255 {{!RISTRETTO=I-D.irtf-cfrg-ristretto255-decaf448}}
   - HashToGroup(): hash_to_ristretto255
-    {{!I-D.irtf-cfrg-hash-to-curve}} with DST:
-    "VOPRF05-" || contextString, where `contextString` is that which is
-    computed in the Setup functions.
+    {{!I-D.irtf-cfrg-hash-to-curve}} with DST
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:923]:
```suggestion
    {{!I-D.irtf-cfrg-hash-to-curve}} with DST =
```
and the same for other suites.

Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md:923]: Can you please turn the others into suggestions?

File: draft-irtf-cfrg-voprf.md [hunk 2]
```diff
@@ -952,8 +954,8 @@ and ristretto255. See {{cryptanalysis}} for related discussion.
 
 - Group: P-256 (secp256r1) {{x9.62}}
   - HashToGroup(): P256_XMD:SHA-256_SSWU_RO\_
-    {{!I-D.irtf-cfrg-hash-to-curve}} with DST:
-    "VOPRF05-" || contextString, where `contextString` is that which is
+    {{!I-D.irtf-cfrg-hash-to-curve}} with DST
+    "VOPRF05-" || `contextString`, where `contextString` is that which is
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:958]: Looking at the .txt output file the use of (') is replaced by quotes. So we might want to remove apostrofes everywhere.

File: draft-irtf-cfrg-voprf.md [hunk 3]
```diff
@@ -936,9 +937,10 @@ and ristretto255. See {{cryptanalysis}} for related discussion.
 
 - Group: decaf448 {{!RISTRETTO}}
   - HashToGroup(): hash_to_decaf448
-    {{!I-D.irtf-cfrg-hash-to-curve}} with DST:
-    "VOPRF05-" || contextString, where `contextString` is that which is
-    computed in the Setup functions.
+    {{!I-D.irtf-cfrg-hash-to-curve}} with DST
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:940]:
```suggestion
    {{!I-D.irtf-cfrg-hash-to-curve}} with DST =
```

File: draft-irtf-cfrg-voprf.md [hunk 4]
```diff
@@ -952,8 +954,8 @@ and ristretto255. See {{cryptanalysis}} for related discussion.
 
 - Group: P-256 (secp256r1) {{x9.62}}
   - HashToGroup(): P256_XMD:SHA-256_SSWU_RO\_
-    {{!I-D.irtf-cfrg-hash-to-curve}} with DST:
-    "VOPRF05-" || contextString, where `contextString` is that which is
+    {{!I-D.irtf-cfrg-hash-to-curve}} with DST
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:957]:
```suggestion
    {{!I-D.irtf-cfrg-hash-to-curve}} with DST =
```

File: draft-irtf-cfrg-voprf.md [hunk 5]
```diff
@@ -967,8 +969,8 @@ and ristretto255. See {{cryptanalysis}} for related discussion.
 
 - Group: P-384 (secp384r1) {{x9.62}}
   - HashToGroup(): P384_XMD:SHA-512_SSWU_RO\_
-    {{!I-D.irtf-cfrg-hash-to-curve}} with DST:
-    "VOPRF05-" || contextString, where `contextString` is that which is
+    {{!I-D.irtf-cfrg-hash-to-curve}} with DST
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:972]:
```suggestion
    {{!I-D.irtf-cfrg-hash-to-curve}} with DST =
```

File: draft-irtf-cfrg-voprf.md [hunk 6]
```diff
@@ -982,8 +984,8 @@ and ristretto255. See {{cryptanalysis}} for related discussion.
 
 - Group: P-521 (secp521r1) {{x9.62}}
   - HashToGroup(): P521_XMD:SHA-512_SSWU_RO\_
-    {{!I-D.irtf-cfrg-hash-to-curve}} with DST:
-    "VOPRF05-" || contextString, where `contextString` is that which is
+    {{!I-D.irtf-cfrg-hash-to-curve}} with DST
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:987]:
```suggestion
    {{!I-D.irtf-cfrg-hash-to-curve}} with DST =
```

File: draft-irtf-cfrg-voprf.md (hunk 2 above)
Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md:958]: Hmm... good catch. I'll change that after merging.

**Review by armfazh — APPROVED** [review]

**Pull request merged**